tests/unit_tests/test_dqc_us_0006.py

```diff
@@ -40,10 +40,26 @@ def test_lea_facts_and_update(self):
         fact1 = Mock(context=context1)
         fact2 = Mock(context=context2)
         fact3 = Mock(context=context2)
-        res1 = src.util.facts.LegalEntityAxis_facts_by_member([fact1, fact2])
-        res2 = src.util.facts.LegalEntityAxis_facts_by_member([fact3])
+        res1 = dqc_us_rules.util.facts.LegalEntityAxis_facts_by_member([fact1, fact2])
+        res2 = dqc_us_rules.util.facts.LegalEntityAxis_facts_by_member([fact3])
         res3 = dqc_us_0006._dict_list_update(res1, res2)
 
         expected = defaultdict(list)
         expected.update({'': [fact2, fact3], 'Company1': [fact1]})
         self.assertEqual(res3, expected)
+
+class Test_Date_Bounds_CVS(unittest.TestCase):
+    def test_date_bounds_cvs_keys(self):
```

Comment:
should add a test that checks that an obviously non-equal dict is not equal to the below expected return